<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labelUnderline">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RightUp">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>85</width>
     <height>86</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RightMid">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>85</width>
     <height>177</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LeftUp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>85</width>
     <height>86</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LeftMid">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>85</width>
     <height>177</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RightButtom">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>85</width>
     <height>177</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LeftButtom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>85</width>
     <height>177</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
